CHANGES.rst

```diff
@@ -8,6 +8,10 @@
 Changes
 =======
 
+Version 1.3.0a5 (released 2020-05-29)
+
+- Fixes issues in Semantic UI `header_login` and `header` templates.
```

Comment:
Comment: Normally we just replace the previous alpha release text (as all the alpha release texts will be replaced by a single version 1.3.0 change log entry).

Don't fix this now in the PR, just merge. We'll fix it the final 1.3.0 release.

Reply:
Ok I see! Noted :)